app/views/listings/index.html.erb

```diff
@@ -38,7 +38,7 @@
     <div class="listing-filters-categories">
       <a href="<%= listings_path %>" class="<%= "selected" if params[:category].blank? %> data-no-instant=" true ">all</a>
       <% categories_for_display.each do |cat| %>
-        <a href="<%= listing_category_path(category: cat[:slug]) %>" class="<%= "selected" if params[:category] == cat[:slug] %> data-no-instant="true"><%= cat[:name] %></a>
+        <a href="<%= listing_category_path(category: cat[:slug]) %> " class="<%= "selected" if params[:category] == cat[:slug] %> data-no-instant="true"><%= cat[:name] %></a>
```

Comment:
weird that it created a space between the tag and the quote

Reply:
I still don't fully-get why this space is necessary to be honest, as erblint isn't complaining for other similar lines 😕

Reply:
I'm not sure why it's needed but I'm definitely guilty of using `--no-verify` to avoid adding it. 😉